es-bs-base/scss/_variables.scss

```diff
@@ -384,13 +386,13 @@ $table-border-level:          -6 !default;
 //
 // Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
 
-$input-btn-padding-y:         .65rem !default;
-$input-btn-padding-x:         .75rem !default;
+$input-btn-padding-y:         0.9rem !default;
+$input-btn-padding-x:         1rem !default;
 $input-btn-font-family:       $font-family-sans-serif !default;
 $input-btn-font-size:         $font-size-base !default;
-$input-btn-line-height:       $line-height-base * 0.9 !default;
+$input-btn-line-height:       1.2 !default;
 
-$input-btn-focus-width:       .2rem !default;
+$input-btn-focus-width:       0 !default;
```

Comment:
Are there any a11y implications to removing addition width on focus? This doesn't affect focus outlines right?

Reply:
Yea I wondered this about a few of these changes regarding focus and a11y. I'm not 100% sure the correct thing but you are right there are still border/color changes for focus.